src/bootstrap/sanity.rs

```diff
@@ -227,14 +227,4 @@ $ pacman -R cmake && pacman -S mingw-w64-x86_64-cmake
     if let Some(ref s) = build.config.ccache {
         cmd_finder.must_have(s);
     }
-
-    if build.config.channel == "stable" {
-        let stage0 = t!(fs::read_to_string(build.src.join("src/stage0.txt")));
-        if stage0.contains("\ndev:") {
-            panic!(
-                "bootstrapping from a dev compiler in a stable release, but \
```

Comment:
Where did this check go to? I'm not seeing the message at a quick grep -- it seems important to make sure we are pinned to static.r-l.o with hashes on stable.

Reply:
Oh, I think this is the conversation we had about not doing dev-static at all anymore, right? Resolving.

Reply:
Yeah, this was removed because we were not planning to use dev-static anymore.